templates/omero-web-config-for-webapps.j2

```diff
@@ -22,3 +22,8 @@ config append -- omero.web.ui.center_plugins '["Auto Tag", "omero_webtagging_aut
 # tagsearch
 config append -- omero.web.apps '"omero_webtagging_tagsearch"'
 config append -- omero.web.ui.top_links '["Tag Search", "tagsearch"]'
+
+# parade
+config append -- omero.web.apps '"omero_parade"'
```

Comment:
This will conflict with https://github.com/openmicroscopy/prod-playbooks/blob/806ab1485d2c4e3586584761f125d5c21a57616a/omero/training-server/files/omero-web-outreach-webapps.omero#L9 (two duplicate web-apps), can you remove the relevant lines from `omero/training-server/files/omero-web-outreach-webapps.omero`?

Reply:
Alternatively we could duplicate omero-web-config-for-webapps.j2 into omero-web-outreach-webapps.omero so outreach only uses one instead of both.

Reply:
Okay, removed 2nd parade install from outreach's file. Thanks @manics